Utility/Framework/SymfonyFramework.php

```diff
@@ -75,19 +78,6 @@ public static function isKernelLessThan($major, $minor = null, $patch = null)
      */
     private static function kernelVersionCompare($operator, $major, $minor = null, $patch = null)
     {
-        $vernum = $major;
-        $kernel = Kernel::MAJOR_VERSION;
-
-        if ($minor) {
-            $vernum .= '.'.$minor;
-            $kernel .= '.'.Kernel::MINOR_VERSION;
-
-            if ($patch) {
-                $vernum .= '.'.$patch;
-                $kernel .= '.'.Kernel::RELEASE_VERSION;
-            }
-        }
-
-        return version_compare($kernel, $vernum, $operator);
+        return version_compare(Kernel::VERSION_ID, sprintf("%d%'.02d%'.02d", $major, $minor ?: 0, $patch ?: 0), $operator);
```

Comment:
Using VERSION instead of VERSION_ID is more in line with what the code previously did, but other projects use VERSION_ID and that works for them so there is some empirical evidence that it works I'd say go for it.

Of course, feature detection (`class_exists`, `method_exists` et al) is always better than kernel version comparison, but that's beside the point of this PR.